oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/PhysicalIndexer.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.storage.plugin.elasticsearch.base;
+
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.skywalking.apm.util.StringUtil;
+import org.apache.skywalking.oap.server.core.storage.model.Model;
+
+/**
+ * The metrics data, that generated by OAL or MAL, would be partitioned to storage by the functions of the OAL or MAL.
+ * And, the other record data would be insulated storage by themselves definitions.
+ */
+@Slf4j
+public enum PhysicalIndexer {
```

Comment:
```suggestion
public enum PhysicalIndices {
```

Typo.

Reply:
There are 2 classes, one is PhysicalIndices, another is PhysicalIndexer.
PhysicalIndices stores the relations between the logic name and the physical name.
PhysicalIndexer is an indexer to help create some meta data.

Reply:
Let's merge 2 into IndexControl, https://github.com/apache/skywalking/pull/6499#discussion_r588883890